release-blocking-jobs.md

```diff
@@ -2,56 +2,122 @@
 
 Here is a list of jobs that should block release if not healthy.
 
-If you want to add a release-branch job, please add them to sig-release-x.x-all dashboard first,
-and show the test is not flaky, then the release team can decide if the test should be added
-to release-blocking dashboard as well.
+If you want to add a release-branch job, please add them to the appropriate
+sig-release-x.y-all dashboard first. Then show the test is not flaky, and the
+release team can decide if the test should be added to release-blocking 
+dashboard as well.
 
-<!-- TODO(krzyzacy): define blocking criteria -->
+## Release Blocking Criteria
+
+The ideal release-blocking job meets the following criteria:
+
+- It provisions clusters via open-source software instead of a hosted service
+<!-- TODO(spiffxp): 
+  - should any allowances be made for conformance?
+-->
+- It finishes a run in 120 minutes or less
+<!-- TODO(spiffxp): 
+  - this should be a percentile, not a max 
+-->
+- It runs at least every 3 hours
+- It passes 75% of all of its runs in the past week (regardless of underlying
+  commit(s))
+- Is capable of passing 3 times in a row against the same commit
+- Is owned by a SIG that is response to addressing failures
+<!-- TODO(spiffxp):
+  - represented by a configured testgrid alert
+  - contact info is also part of job description
+-->
+- Fails for no more than 10 runs in a row
+- Has a documented reason for its inclusion in the release-blocking suite
+<!-- TODO(spiffxp):
+  - represented in description? link to an issue where discussion was had?
+-->
+
+The reasoning for the above is that as we get to the end of a release cycle,
+we are usually waiting on a handful of bug fixes and want to be certain that
+the underlying bug is truly fixed. This means we prefer to see multiple test
+results for the same commit. If we wait for three successful runs that are
+scheduled every three hours, we wait for nine hours.  This effectively gives
+us the chance to make two changes/decisions in a 24 hour period.
+
+The thresholds given are aspirational, and we recognize that not every job may
+be able to strictly adhere to all of these criteria all of the time. Thus we
+consider human review a key component in both the addition and removal of a
+job's release-blocking status.
+
+We intend to drive the criteria to stricter thresholds over time, e.g.,
+- finishing a run in 60min or less
+- running at least every 2 hours
+- passing at least 90% of all runs1
+
+## Release Informing Criteria
+
+Jobs that are considered useful or necessary to inform whether a commit on
+master is ready for release, but that clearly do not meet release-blocking 
+criteria, may be placed on the sig-release-master-informing dashboard. These
+are often jobs that are some combination of slow, flaky, infrequent, or
+difficult to maintain.
+
+These jobs may still block the release, but because they require a lot of
```

Comment:
I like the idea of moving the "noisy" jobs into a separate dashboard so we dint loose their signal right away and help maintain the "blocking" dashboard in a green state for most part. But I am afraid we will still end up spending the same amount of time and effort chasing the failures in the "informing" dashboard to determine  GO/NoGO for a release. I am not sure when  and how much to dial down our attention towards the failures here.
It might be worthwhile to add some kind of ownership expectation for these jobs eg:  if we dont get attention from Job/Test Owners around failing tests within 'x' days we will not block release on them.

Reply:
I'm hoping if nothing else it means we spend our time babysitting fewer jobs.  These jobs should still have clear owners etc. But I think our decision to block/not-block is the "human interpretation" part I'm trying to call out here.  I'll see if I can re-word